modules/apps/object/object-test/src/testFunctional/macros/Picklist.macro

```diff
@@ -137,6 +137,19 @@ definition {
 			locator1 = "Picklist#DISABLE_PICKLIST_KEY");
 	}
 
+	macro assertDownloadedPicklist {
```

Comment:
I've tried many macros, and none would work because of the random number that comes after ListType_{newPicklistName}. The only way that this test would work was by using my new macros. I asked the guys on my team for help, and they wouldn't find macros that would work on this specific test, so I figured out that the best option was to create this macro.

Reply:
Ah okay, sorry I didn't know about that. Thank you for the explanation. In that case, the macro and naming should be good. @davicms